lib/src/scss-shared/_nx-focus-border-helpers.scss

```diff
@@ -15,3 +15,10 @@
     height: calc(#{$height} + 6px);
     width: calc(#{$width} + 6px);
 }
+
+@mixin hide-checkbox {
```

Comment:
This doesn't really have to do with focus, I'd rather it be in its own file since I don't think it fits cleanly in any of the existing ones.